app/config/frontendAppConfig.scala

```diff
@@ -152,6 +154,9 @@ class FrontendAppConfig @Inject()(val runModeConfiguration: Configuration, envir
   override val agentServicesHost: String = sc.getString(ConfigKeys.agentServicesHost)
   override val agentServicesUrl: String = agentServicesHost + sc.getString(ConfigKeys.agentServicesUrl)
 
+  override val agentClientServicesHost: String = sc.getString(ConfigKeys.agentClientHost)
+  override val agentClientServicesUrl: String = agentClientServicesHost + sc.getString(ConfigKeys.agentClientServicesUrl)
```

Comment:
Can you remove lines 75 and 157 and simply update your URL value to be `sc.getString(host) + sc.getString(url)`? Looks unnecessary to me to make the host accessible like this when it's not being reused for any other purposes.